examples/nextjs/app/workflow/route.ts

```diff
@@ -0,0 +1,40 @@
+import { Client } from "@upstash/qstash";
+
+const client = new Client({
+  baseUrl: process.env.QSTASH_URL!,
+  token: process.env.QSTASH_TOKEN!
+});
+
+export async function POST(request: Request) {
+  
+  // hide the workflow url
+  const context = await client.workflow(request)
+
+  // {value : 4}
+  const result1 = await context.run("step uno", () => {
+    console.log("A");
+    return Promise.resolve({value : 2 + 2});
+  });
+
+  const now = new Date()
+  now.setSeconds(now.getSeconds() + 5);
+  await context.sleepUntil("sleeping", now)
```

Comment:
I assume you need to pass a string for us to show this sleep with a corresponding name in the dashboard, from user perspective this doesn't seem intuitive though. I suggest just `context.sleepUntil(inFiveSeconds)` and using i.e. the current url path where this sleep is invoked as the string id

Reply:
we wanted to have names for each step, sleeps included

url is not a good alternative I think, there can be several sleep statements